paper/paper.md

```diff
@@ -46,59 +46,59 @@ bibliography: paper.bib
+1. **Repetitive and Error-Prone Setups**: DL typically involves significant boilerplate code for training loops, data loading, and metric calculations. The numerous hyperparameters and components across experiments can easily become complex and error-prone. Lighter abstracts these repetitive tasks and uses centralized configs for a clear, manageable experimental setup, reducing tedium and potential for errors.
 
-2.  **Experiment Management:** Handling numerous hyperparameters and configurations across various experiments can become cumbersome and error-prone. *Lighter offers organized configuration through YAML files, providing a **centralized record of all experiment parameters**.*
+2. **Reproducibility and Collaboration**: Inconsistent or complex codebases hinder collaboration and experiment reproduction. Lighter's self-documenting configs offer clear, structured snapshots of each experiment. This greatly improves reproducibility and simplifies how teams share and reuse setups.
 
-3.  **Reproducibility:** Reproducing experiments from different implementations can be challenging. *Lighter's **self-contained configuration files** serve as comprehensive documentation, facilitating the exact recreation of experimental setups.*
+3. **Pace of Research Iteration**: The cumulative effect of these challenges inherently slows down the research cycle. Lighter streamlines the entire experimental process, allowing researchers to focus on core hypotheses and iterate on ideas  efficiently.
 
-4.  **Collaboration:** Collaborating on experiments often requires understanding complex codebases. *Lighter enhances collaboration by using standardized configurations, making it easier to share and reuse experiment setups within and across research teams.*
-
-5.  **Slowed Iteration:** The cumulative effect of these challenges slows down the research iteration cycle. *Lighter accelerates iteration by streamlining the experiment setup process, allowing researchers to focus on core experiment choices without being bogged down by infrastructure concerns.*
+# State of the Field
 
+Config-driven frameworks like Ludwig [@Ludwig], Quadra [@Quadra], and GaNDLF [@Gandlf] provide high levels of abstraction by encapsulating all components within predefined structures. While this approach simplifies usage, it limits flexibility to modify the flow or extend components, often requiring direct source code changes.
+Lighter takes a different approach by providing medium-level abstraction. It implements a unified flow while maintaining direct compatibility with standard PyTorch components (models, datasets, optimizers). The flow itself is modifiable to any task via [adapters](#adapters), while custom code is [importable via config](#project-specific-modules) without source code modifications.
```

Comment:
> It implements a unified flow while maintaining direct 

Unified flow again sounds odd.

How about
"It implements a structured pipeline maintaining direct compatibility ..... The pipeline extends to any task via adapters while custom code is ...."

Reply:
Maybe workflow is better too